terraform/gcp/modules/ctlog/main.tf

```diff
@@ -37,4 +42,50 @@ resource "google_dns_record_set" "A_ctfe" {
   managed_zone = var.dns_zone_name
 
   rrdatas = [var.load_balancer_ip]
-}
\ No newline at end of file
+}
+
+// Private network
```

Comment:
i dont think this is necessary, as we dont want to create yet another private network.
typically we only need one per environment, and the network config can be passed in as vars.